ouroboros-consensus-cardano-test/src/Test/Consensus/Cardano/MockCrypto.hs

```diff
@@ -1,45 +1,50 @@
-{-# LANGUAGE DataKinds    #-}
-{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE DataKinds         #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TypeFamilies      #-}
 {-# OPTIONS_GHC -Wno-orphans #-}
 module Test.Consensus.Cardano.MockCrypto (
     BlockCompatByron
-  , TPraosMockCryptoCompatByron
+  , MockShelleyCompatByron
+  , MockCryptoCompatByron
   ) where
 
 import           Cardano.Crypto.DSIGN (Ed25519DSIGN)
-import           Cardano.Crypto.Hash (Blake2b_224, HashAlgorithm)
+import           Cardano.Crypto.Hash (Blake2b_224, Blake2b_256)
 import           Cardano.Crypto.KES (MockKES)
 
+import           Cardano.Ledger.Crypto (Crypto (..))
+import           Shelley.Spec.Ledger.BaseTypes (Shelley)
 import           Test.Cardano.Crypto.VRF.Fake (FakeVRF)
 
 import           Ouroboros.Consensus.Shelley.Ledger (ShelleyBlock)
 import           Ouroboros.Consensus.Shelley.Protocol.Crypto (TPraosCrypto)
 
-import           Shelley.Spec.Ledger.Crypto (Crypto (..))
-
--- | A mock replacement for 'TPraosStandardCrypto' that is compatible with
+-- | A mock replacement for 'StandardCrypto' that is compatible with
 -- bootstrapping from Byron
 --
 -- This mocks more components than does
```

Comment:
I think this first sentence is either out-of-date and/or was wrong to begin with. `MockCryptoCompatByron` does _not_ mock more than `MockCrypto` does.

Also, the last item in the commit message 6fdeac5633fe2034080ccbe240c5369fdd604bb6 of says that the generators can be re-used, so the last two sentences are also out-of-date, if I understand correctly.

Reply:
Fixed, thanks.